creator-node/src/routes/files.js

```diff
@@ -123,6 +134,9 @@ const streamFromFileSystem = async (
         })
     })
   } catch (e) {
+    // Unset the cache-control header so that a bad response is not cached
+    res.removeHeader('cache-control')
```

Comment:
idt we need to remove `Content-Disposition` here, cus originally this header never got removed on 404 either